src/Storage/Storage.Management/Blob/UpdateAzureStorageBlobServiceProperties.cs

```diff
@@ -0,0 +1,148 @@
+    {
+        /// <summary>
+        /// AccountName Parameter Set
+        /// </summary>
+        private const string AccountNameParameterSet = "AccountName";
+
+        /// <summary>
+        /// Account object parameter set 
+        /// </summary>
+        private const string AccountObjectParameterSet = "AccountObject";
+
+        /// <summary>
+        /// BlobServiceProperties ResourceId  parameter set 
+        /// </summary>
+        private const string PropertiesResourceIdParameterSet = "BlobServicePropertiesResourceId";
+
+        /// <summary>
+        /// AccountName Parameter Set
+        /// </summary>
+        private const string AccountNamePropertiesObjectParameterSet = "AccountNamePropertiesObject";
+
+        [Parameter(
+          Position = 0,
+          Mandatory = true,
+          HelpMessage = "Resource Group Name.",
+         ParameterSetName = AccountNameParameterSet)]
+        [Parameter(
+          Position = 0,
+          Mandatory = true,
+          HelpMessage = "Resource Group Name.",
+         ParameterSetName = AccountNamePropertiesObjectParameterSet)]
+        [ResourceGroupCompleter]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(
+            Position = 1,
+            Mandatory = true,
+            HelpMessage = "Storage Account Name.",
+           ParameterSetName = AccountNameParameterSet)]
+        [Parameter(
+            Position = 1,
+            Mandatory = true,
+            HelpMessage = "Storage Account Name.",
+           ParameterSetName = AccountNamePropertiesObjectParameterSet)]
+        [ResourceNameCompleter("Microsoft.Storage/storageAccounts", nameof(ResourceGroupName))]
+        [Alias(AccountNameAlias)]
+        [ValidateNotNullOrEmpty]
+        public string StorageAccountName { get; set; }
+
+        [Parameter(Mandatory = true,
+            HelpMessage = "Storage account object",
+            ValueFromPipeline = true,
+            ParameterSetName = AccountObjectParameterSet)]
+        [ValidateNotNullOrEmpty]
+        public PSStorageAccount StorageAccount { get; set; }
+
+        [Parameter(
+            Position = 0,
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Blob Service Properties Resource Id.",
+           ParameterSetName = PropertiesResourceIdParameterSet)]
+        [ValidateNotNullOrEmpty]
+        public string ResourceId { get; set; }
+
+        [Parameter(Mandatory = false, HelpMessage = "Default Service Version to Set")]
+        [ValidateNotNull]
+        public string DefaultServiceVersion { get; set; }
+
+        [Parameter(Mandatory = true, HelpMessage = "Blob service properties object to Set", ParameterSetName = AccountNamePropertiesObjectParameterSet, ValueFromPipeline = true)]
+        [ValidateNotNullOrEmpty]
+        public PSBlobServiceProperties BlobServiceProperty { get; set; }
```

Comment:
Shouldn't this be the parameter that takes its value from the pipeline?  In that case, the verb for this cmdlet should eb 'Set'.  'Update' only makes sense if you are using PATCH semantcis, meaning that you can set one blob service property and leave others unchanged.

Reply:
We can set one blob service property and leave others unchanged, so "Update" might be more proper. 
Actually, this is also aligned with dataplane cmdlet name: Update-AzureStorageServiceProperties. And "Update" verb is suggest by you when design the dataplane cmdlet

Reply:
As we talked, I have removed parameter "PSBlobServiceProperties BlobServiceProperty" from this cmdlet. We might can add a Set-* cmdlet for this in the future, but not in this release.